packages/svelte-docgen/src/doc/type.ts

```diff
@@ -2,181 +2,179 @@ import type { extract } from "@svelte-docgen/extractor";
 
 import type { TypeKind } from "./kind.js";
 
-export declare namespace Doc {
```

Comment:
ESLint says “don’t use namespaces”

Reply:
That's alright, you have already implemented a workaround with:

```js
import * as Doc from "./types.ts";
```